src/main.py

```diff
@@ -147,7 +149,7 @@ def _run_pipeline(env: BaseEnv,
                 continue
             logging.info(f"\n\nONLINE LEARNING CYCLE {i}\n")
             logging.info("Getting interaction requests...")
-            if total_num_transitions > CFG.online_learning_max_transitions:
+            if num_online_transitions > CFG.online_learning_max_transitions:
```

Comment:
@amburger66 note: we're capping _online_ transitions now, so we're going to be running the scripts for longer. I think this is more intuitive?

Reply:
makes sense